automation/README.md

```diff
@@ -0,0 +1,86 @@
+# Automation with oVirt CI
+
+In order to keep playbooks in this repository operational,
+it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
+Everything located under `automation` directory is related to integration
+to [oVirt CI System][ovirt-ci-system-doc].
+
+There is a list of events which can trigger a job on [oVirt CI System][ovirt-ci-system-doc],
+The KubeVirt Ansible repository uses the `check-patch` event at the moment,
+to verify incoming patches.
+
+Once you submit a PR for this repository, the [oVirt CI System][ovirt-ci-system-doc]
+triggers [Jenkins job](http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity)
+which executes the [`automation/check-patch.sh`](./check-patch.sh) script.
+
+This script will execute the Ansible playbook [`./playbooks/automation/check-patch.yml`](../playbooks/automation/check-patch.yml)
+which wraps entire testing flow.
+
+
+## `check-patch.yml` Ansible playbook
+
+This Ansible playbook is a wrapper for the entire testing flow which is composed from
+following steps (partial playbooks)
+* [provision testing environment](#testing-environment)
+* [deploy OpenShift](#deploy-openshift)
+* [install KubeVirt](#install-kubevirt)
+
+This playbook is executed inside of CentOS 7.4 mock.
+There is additional software installed, please read
+[automation/check-patch.packages](./check-patch.packages) for complete list.
+
+Parameters and usage of this playbook is described at
+[Deploy new Kubernetes or OpenShift cluster and KubeVirt with Lago](../README.md#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago).
+
+This is list of playbooks which are executed.
+* [check-patch.yml](../playbooks/automation/check-patch.yml)
+  * [playbooks/provider/lago/config.yml](../playbooks/provider/lago/config.yml)
+  * [playbooks/cluster/openshift/config.yml](../playbooks/cluster/openshift/config.yml)
+  * [playbooks/components/install-kubevirt-release.yml](../playbooks/components/install-kubevirt-release.yml)
+
+### Testing environment
+
+To provision testing environment the [`playbooks/provider/lago/config.yml`](../playbooks/provider/lago/config.yml) playbook is used.
+
+Testing environment is populated by
+[Lago](https://github.com/lago-project/lago) project, it provisions desired
+resources to match requirements described in Lago configuration file.
+This configuration file is located in root of repository
+[`LagoInitFile.yml`](../LagoInitFile.yml).
```

Comment:
This file was also moved to `playbooks/provider/lago/`

Reply:
addressed